sql/core/src/test/scala/org/apache/spark/sql/test/DataFrameReaderWriterSuite.scala

```diff
@@ -428,7 +428,7 @@ class DataFrameReaderWriterSuite extends QueryTest with SharedSQLContext with Be
     val message = intercept[AnalysisException] {
       testRead(spark.read.csv(), Seq.empty, schema)
     }.getMessage
-    assert(message.toLowerCase(Locale.ROOT).contains("unable to infer schema for csv"))
+    assert(message.contains("Unable to infer schema for CSV. It must be specified manually."))
```

Comment:
why remove `toLowerCase(Locale.ROOT)`?

Reply:
Because both V1 and V2 show exactly the same message. 
`toLowerCase(Locale.ROOT)` was added in the migration of CSV V2 https://github.com/apache/spark/pull/24005/files#diff-b9ddfbc9be8d83ecf100b3b8ff9610b9R431